docs/apis/phi-silica.md

```diff
@@ -64,6 +64,23 @@ var result = await languageModel.GenerateResponseAsync(prompt);
 Console.WriteLine(result.Response); 
 ```
 
+```cpp
```

Comment:
@Karl-Bridge-Microsoft is there a way to collapse the C# and C++ samples together with a toggle for the developer to see their chosen one at a time? I've seen this in other docs.

Reply:
@jwmsft since Karl is OOF

Reply:
add this metadata to the top of the page after the author information:
```
dev_langs:
- csharp
- cpp
```

That will add a language switch at the top of the page to let the user choose which language they want to show.